### ISSUE TITLE
RMD publication record contains the former name of the journal that published the article

### ISSUE THREAD

**Issue by anaelizabethenriquez** [psu-libraries/researcher-metadata#948]
Allie, who curates ScholarSphere deposits, has noticed several deposits where the journal title is incorrect, and it is an earlier name for the journal. She has corrected these as part of her ScholarSphere curation work.

The errors seem to be coming from RMD. [Publication 514564](https://metadata.libraries.psu.edu/admin/publication/514564) is an example. RMD lists the journal as "[Journal of the Multihandicapped Person](https://metadata.libraries.psu.edu/admin/journal/22799)," but the current name (since 1991) is "Journal of Developmental and Physical Disabilities." 514654 has two associated imports, one from Pure and one from Activity Insight. The journal name in Pure is currently correct. I can't check Activity Insight, but I suspect it's correct there too, as this journal only had the earlier name for a few years (it started in 1988), long ago.

Could this be related to how RMD stores journals as items in the database? 

Ideally RMD would use the name that the journal had at the time a given article was published.

**Comment by anaelizabethenriquez** [comment]
Today I ran across this publication, where the correct journal title is listed in the text field for the journal, but there is an incorrect journal linked to the record: https://metadata.libraries.psu.edu/admin/publication/286215

I can't seem to remove the incorrect journal, and I wonder if this is related to the issue here.

**Comment by ajkiessl** [comment]
@anaelizabethenriquez Both of these, and the one in https://github.com/psu-libraries/researcher-metadata/issues/984, have the same problem.  Pure stores a handful of titles within the journal object.  Here's what the data looks like in Pure's backend for these three:

![Image](https://github.com/user-attachments/assets/5474fd06-096d-4af3-9ae2-987051bc5678)
--------------------------------------------------------------------------
![Image](https://github.com/user-attachments/assets/a0b0f350-c8e6-4f81-bfcc-1e00bf7e9903)
--------------------------------------------------------------------------
![Image](https://github.com/user-attachments/assets/b0cbafb6-f931-47e4-a497-9b087e5e9c28)


When importing these journals, we just select the first title, which is wrong in these cases.  Interestingly for two of the above, the alternative title is correct.  However,  "Journal of the Multihandicapped Person" does not have an alternative title.  This _seems_ like a data integrity problem in Pure, but it might just be how they want to store titles.  I'd need more context around why they store titles this way and why sometimes the same title is stored twice.

The actual title stored with the publication (research-output) in Pure, which is displayed in the UI, is selected from these "titles" with the journal object.  I'm not sure how it's selected, though.  In the backend, the publications are linked to the correct journals.  I'm assuming whatever title is chosen is the title of the journal when the publication was published.

I believe the only way to fully solve this, and make sure the journal title is the title of that journal when the publication was published, is to store the journal title associated with the research-output in Pure in the journal_title field associated with publications in RMD.  Then, change our preferred journal title logic to give preference to that journal_title field over the title associated with the journal object in RMD.  The journal_title field holds the journal title from non-Pure sources.  So, we'd have to consider conflicts.  Also, we'd need to change some of our merging logic.

It does seem like the data is a bit messy in Pure, though, and maybe they could clean it up so the most recent title is the first title in those "titles" lists.  That would be simpler for us, and mostly fix this.  We wouldn't be handling the different titles at different publish times, though.  We'd just have the most recent title for that journal.

**Comment by ajkiessl** [comment]
>> Do you have a path for reporting problems with the data structure?

No, but I'll look into it.  I think it's less of a structural problem and moreso just messy data.  I could understand them storing titles chronologically. Then, all we'd need to do is grab the last one for the most recent.  But, there's duplicates in there and it goes from the old title, to the new title, then back to the old for some.  I'll also see if there's any documentation on what gets stored here.

>> A lot of publications that are journal articles don't have an associated "journal" record, right? And it sounds like the journals that are objects get more deference in merging that journal titles that are just in a field. Is that right? Any idea what the argument for that was?

Yeah, about a third of them don't have an associated "journal" record.  And yes, if there's an associated journal record it is given deference in merging.  We just get rid of anything that was in the `journal_title` field if there's an associated journal record.  I think the argument was that since the associated journal record is always from Pure, that data is more reliable than the other sources.

**Comment by ajkiessl** [comment]
I emailed the Pure team on 12/16.  No response.

**Comment by anaelizabethenriquez** [comment]
@djs7648 If this gets fixed and you remember it, it would be great to follow up in this LibAnswers ticket: https://psu.libanswers.com/admin/ticket?qid=14812218

**Comment by anaelizabethenriquez** [comment]
Another LibAnswers ticket to follow up with if we manage to fix this: https://psu.libanswers.com/admin/ticket?qid=13315853
